modules/account-lib/test/unit/coin/hbar/transaction.ts

```diff
@@ -7,6 +7,9 @@ import { KeyPair } from '../../../../src/coin/hbar/keyPair';
 describe('Hbar Transaction', () => {
   const coin = coins.get('thbar');
 
+  /**
+   *
+   */

```

Comment:
Remove empty doc

Reply:
It looks like the auto-lint is adding these lines to any declared function. We are having the same issue on every transaction test class (if we check celo, etc, or any other we have the same issue.) 
@twtaylorbitgo what do you think? Maybe we can add a cleanup ticket to clean these empty jsdoc and avoid using a function inside of the tests files.